src/SWP/Bundle/CoreBundle/DependencyInjection/Configuration.php

```diff
@@ -70,6 +74,26 @@ public function getConfigTreeBuilder()
                                             ->scalarNode('object_manager_name')->defaultValue(null)->end()
                                         ->end()
                                     ->end()
+                                    ->arrayNode('facebook_instant_articles_feed')
+                                        ->addDefaultsIfNotSet()
+                                        ->children()
+                                            ->scalarNode('model')->cannotBeEmpty()->defaultValue(FacebookInstantArticlesFeed::class)->end()
+                                            ->scalarNode('interface')->defaultValue(FacebookInstantArticlesFeedInterface::class)->end()
+                                            ->scalarNode('repository')->defaultValue(EntityRepository::class)->end()
+                                            ->scalarNode('factory')->defaultValue(Factory::class)->end()
+                                            ->scalarNode('object_manager_name')->defaultValue(null)->end()
+                                        ->end()
+                                    ->end()
+                                    ->arrayNode('facebook_instant_articles_article')
```

Comment:
maybe let's name it `fbia_article` because the services names will be very long in this case, e.g.: `swp.repository.facebook_instant_articles_article` and it can be simplified to `swp.repository.fbia_article`